pkg/actuators/client/client.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021.
+Copyright 2021, 2022.
```

Comment:
We conventionally just leave these as the year the file was created rather than updating them, bit of a K8s community convention

Reply:
Okay, I will revert those changes then quick.

Reply:
reverted